core/src/main/resources/application.conf

```diff
@@ -103,10 +103,12 @@ workflow-options {
   //workflow-failure-mode: "ContinueWhilePossible"
 }
 
-// Optional call-caching configuration. Disabled by default.
+// Optional call-caching configuration.
 call-caching {
   enabled = false
 
+  // This option determines whether the name of a docker image should be hashed (if docker is specified for a task)
+  hash-docker-names = true
```

Comment:
To be clear about what I was advocating for, I think we should *always* hash the docker tuple and file paths. By default we also hash from dockerhub and the file hash themselves, but one can disable that. If we have the dockerhub and/or the file hash we'll only use those, not the "simple" variants (i.e. if i'm checking full file hashes I don't care if the path isn't the same)